app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+
+    for dict_person in people:
+        if dict_person.get("husband"):
+            Person.people[dict_person["husband"]].wife = (
+                Person.people)[dict_person["name"]]
+        elif dict_person.get("wife"):
+            Person.people[dict_person["wife"]].husband = (
+                Person.people)[dict_person["name"]]
+        continue
```

Comment:
why do you need continue? Loop automatically iterates through each object in the collection. Continue is used when you need to skip some object like
```
for person in people:
    if person['is_married']:
        continue

```

Reply:
> why do you need continue? Loop automatically iterates through each object in the collection. Continue is used when you need to skip some object like
> 
> ```
> for person in people:
>     if person['is_married']:
>         continue
> ```

most likely this operator simply overlooked it when reworking the previous solution :(